src/animation.cc

```diff
@@ -2787,7 +2787,9 @@ void _object_animate()
 
         Object* object = sad->obj;
 
-        unsigned int time = getTicks();
+        // TODO janiczek: I needed to patch this one to get the speedup when moving etc.
+        // But why doesn't sfall patch it (or if it does, where is it in the offset list)?
```

Comment:
It's 0x4C9375, inside a wrapper function for GetTickCount. That wrapper function is what's called here in the original game code.